Procfile

```diff
@@ -1,2 +1,2 @@
-release: cd network-api && python ./manage.py migrate --no-input
+release: cd network-api && python ./manage.py migrate --no-input && python ./manage.py block_inventory --no-input
```

Comment:
This `--no-input` flag breaks the staging deploy. I've filed cfpb/wagtail-inventory#15 to see if the maintainer of the library is interested in adding support for this flag, and hopefully that results in a new version that lets us add this plugin as part of the automated deployment.

Reply:
So this can't be reviewed until then?